src/cmd/ksh93/edit/edit.c

```diff
@@ -1027,11 +1027,6 @@ void	ed_putchar(Edit_t *ep,int c)
 		size = 1;
```

Comment:
Since the `size` variable won't be referenced further in this function now, this assignment can also be deleted, along with the surrounding braces.

Reply:
Thanks!